packages/babel-plugin-transform-classes/src/loose.js

```diff
@@ -1,6 +1,14 @@
 import nameFunction from "@babel/helper-function-name";
 import VanillaTransformer from "./vanilla";
-import { types as t } from "@babel/core";
+import { template, types as t } from "@babel/core";
+
+const objectDefineProperty = template(`
+  Object.defineProperty(CLASS_NAME, KEY, {
+    KIND: FUNCTION_BODY,
+    enumerable: true,
```

Comment:
This should be `false` (which is the defautl value, so you can just omit it):

```js
> class A { static method() {} }
undefined

> Object.getOwnPropertyDescriptor(A, "method");
{value: ƒ, writable: true, enumerable: false, configurable: true}
```

Reply:
This is missing `writable: true`, though.

However, you can't specific `writable: true` if the descriptor is a getter or setter descriptor...

Also, getters/setters need to be put together in the _same_ descriptor. This would create separate descriptors if both are present so the one to be evaluated last would be the only one remaining.

Reply:
That is the output from the console, I don't think that there should be `writable: true`.

Reply:
Yeah, we can't actually add `writable: true` here, because it's a setter/getter descriptor. It'll throw unless it's a value descriptor.

Reply:
@Kovensky are you sure that last descriptor overrides previous ones? Expanding on @nicolo-ribaudo example:
```js
> class A { static get prop() {} }
undefined

> Object.getOwnPropertyDescriptor(A, "prop");
{get: ƒ, set: undefined, enumerable: false, configurable: true}

> Object.defineProperty(A, 'prop', { set(val) { this.val = val }, enumerable: false, configurable: true })
class A { static get prop() {} }

> Object.getOwnPropertyDescriptor(A, "prop")
{get: ƒ, set: ƒ, enumerable: false, configurable: true}
```

It seems that descriptors are actually merged rather than replacing itself.

Reply:
Cool, I didn't know it!

Reply:
Me neither, still aint sure if that's the case for all edge cases. I have just tested it in Chrome's console 😅

Reply:
Oh, [it's true](https://www.ecma-international.org/ecma-262/5.1/#sec-8.12.9). I wasn't sure so I tried checking the ES5 spec too, and it seems that it does try to merge the descriptors as well. https://www.ecma-international.org/ecma-262/5.1/#sec-8.12.9. I couldn't find an ES 5.0 spec but I guess it's safe to assume it works the same and will merge descriptors as long as they are configurable.